qt/python/mantidqt/widgets/samplelogs/test/test_samplelogs_model.py

```diff
@@ -134,6 +134,24 @@ def test_model_MD(self):
         self.assertEqual(values[2], 1455001)
         self.assertEqual(values[3], "")
 
+    def test_Invalid_data_logs(self):
+        ws = Load('ENGINX00228061_log_alarm_data.nxs')
```

Comment:
It would be better to avoid files where possible and use mocks as any file loading end up affecting the test timings quite badly. I see the rest of the test uses files so I can let this through on this occasion.